lib/shopify_api/rest.ex

```diff
@@ -11,25 +11,74 @@ defmodule ShopifyAPI.REST do
   alias ShopifyAPI.JSONSerializer
   alias ShopifyAPI.REST.Request
 
-  @doc false
-  def get(%AuthToken{} = auth, path, params \\ []) do
-    Request.perform(auth, :get, path, "", params)
+  @doc """
+  Underlying utility retrieval function. The options passed affect both the
+  return value and, ultimately, the number of requests made to Shopify.
+  Options:
```

Comment:
Let's break this out as a header:

```suggestion

  ## Options

```